CHANGELOG.md

```diff
@@ -2090,13 +2090,15 @@ If you see any broken links or other issues, please report them in [#922](https:
 - Prevent rendering Component tags during fill discovery stage to fix a case when a component inside the default slot
   tried to access provided data too early.
 
-## 🚨📢 v0.110
+## v0.110 🚨📢
+
+_25 Nov 2024_
```

Comment:
Added date to older releases. When the date is included, in the format like `_25 Nov 2024_`, then the script that prepares the Releases section in the documentation will pick up on these dates, and append them to the release page title, so e.g. `v0.110.0` becomes `v0.110.0 (2024-11-25)`.



<img width="903" height="815" alt="Screenshot 2025-08-16 at 18 08 38" src="https://github.com/user-attachments/assets/89847c10-8f27-4047-b566-a0795a1d3521" />